client/src/components/Info.style.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  display: flex;
+  flex-direction: column;
+  background-color: var(--white);
+  height: 100vh;
```

Comment:
I think this might work better as a different unit of measurement, or closer to something like ```80vh```. Since the ```header``` takes up some space above the ```main``` tag, it doesn't display properly, since this property makes the whole page bigger than it needs to be